documentation/eslint.md

```diff
@@ -0,0 +1,74 @@
+# Development documentation (Third Iteration)
```

Comment:
Not sure of understanding the purpose of this file...

Apart from the title (what is "third iteration"? maybe it makes sense in the context of fiQare project, but not this repo is project-agnostic), this seem to be just a description of eslint based on standard content.

I mean, I could understand a replacmente of jslint with eslint, as for instance is done by PR https://github.com/telefonicaid/iotagent-ul/pull/408 in the case of IOTA-UL (although not in the scope of this PR but in a new one). However just including a piece of documentation about eslint in the repository doesn't have too much sense to me.

Maybe I'm not getting the point of something. Could you elaborate on this? If it is easier for you, you can tell me by email.

Reply:
This is a style guide made for the Fiqare project, if it is not useful for Perseo Front or for future developers we remove it from the PR.

Reply:
This is a style guide made for the Fiqare project, if it is not useful for Perseo Front or for future developers we remove it from the PR.

Reply:
Yes, it seems to be too much fiQare-specific... Removing it from the PR sounds good to me.

Reply:
Fixed in ce583be19f1561114b48d1dc71d91e661e2e1b11